assets/components/content-types/event/event.twig

```diff
@@ -6,16 +6,16 @@
     </picture>
     {% endblock %}
 
-    <h3 class="card-title">{% block title %}Startup Acceleration Workshops{% endblock %}</h3>
+    <h3 class="card-title event-title" itemprop="name">{% block title %}Startup Acceleration Workshops{% endblock %}</h3>
```

Comment:
Same comment here, we have a nested `Place` schema, but no defined `Event` schema...

Reply:
yes right 903e564